po/vi.po

```diff
@@ -6394,25 +6424,37 @@ msgid ""
 "From%3C%5B(K,+V);+N%5D%3E-for-HashMap%3CK,+V,+RandomState%3E), which allows "
 "us to easily initialize a hash map from a literal array:"
 msgstr ""
+"Thay vào đó, kể từ phiên bản Rust 1.56, HashMap implement [`From<[(K, V); "
+"N]>`](https://doc.rust-lang.org/std/collections/hash_map/struct.HashMap."
+"html#impl-From%3C%5B(K,+V);+N%5D%3E-for-HashMap%3CK,+V,+RandomState%3E), cho "
+"phép chúng ta dễ dàng khởi tạo một hashmap từ một mảng giá trị:"
 
 #: src/std-types/hashmap.md
 msgid ""
 "Alternatively HashMap can be built from any `Iterator` which yields key-"
 "value tuples."
 msgstr ""
+"HashMap cũng có thể được khởi tạo từ bất kỳ `Iterator` nào trả về các cặp "
```

Comment:
Is there any reason for not translating HashMap and Iterator?

Reply:
Ditto.